frontend/public/components/utils/headings.tsx

```diff
@@ -129,12 +130,18 @@ export const PageHeading = connectToModel((props: PageHeadingProps) => {
               {kind && <ResourceIcon kind={kind} className="co-m-resource-icon--lg" />}{' '}
               <span data-test-id="resource-title" className="co-resource-item__resource-name">
                 {resourceTitle}
+                {hasData && data.metadata.ownerReferences && (
+                  <CSVManager className="hidden-lg hidden-xl" obj={data} />
```

Comment:
We should see if we can position this using CSS so we don't need to `CSVManager` twice in the markup.

cc @sg00dwin @rhamilto

Reply:
I couldn't figure out a better way to do it given how design wanted everything to wrap. I'd appreciate any insight you might have @sg00dwin and @rhamilto.

Reply:
I'm wondering if the owner reference should always sit below the resource name?  We already have status sitting beside.  Trying to get the layout right with two possible "badges", one of which has a variable length name, is going to be challenging.  Status alone can be a bit wonky at times.

Reply:
@itsptk - Would you be ok with this? It would lead to cleaner code on our end, but it does change your design a lot.

Reply:
I recall talking with @tlwu2013 about this early on. Initially I had designed this with the managed label beneath the resource name but Tony mentioned it would be hard to retain the same spacing beneath the Action dropdown.

After talking with @rebeccaalpert it does sound like this space would increase some, but only when the label appears. Here are images of without label and with.

Without:
![noLabel](https://user-images.githubusercontent.com/54860125/88297832-6818b380-ccce-11ea-85f3-b44926ecf7d6.png)

With:
<img width="1047" alt="withLabel" src="https://user-images.githubusercontent.com/54860125/88297839-6b13a400-ccce-11ea-9f1d-6c168cd2ae18.png">

I think my preference would still be to have the label to the right, but understand if this makes the code a lot more complicated, that maybe it should just always go beneath. The fact that this larger spacing only happens when the label actually appears makes it more acceptable to me.